handler.go

```diff
@@ -613,15 +611,10 @@ func (h *handler) outOfBandSync(w http.ResponseWriter, r *http.Request) error {
 
 	pathAndParams := r.URL.String()
 
-	deployType := sdk.DeployTypePing
-	if h.useConnect {
-		deployType = sdk.DeployTypeConnect
-	}
-
 	config := sdk.RegisterRequest{
 		URL:        fmt.Sprintf("%s://%s%s", scheme, host, pathAndParams),
 		V:          "1",
-		DeployType: deployType,
+		DeployType: sdk.DeployTypePing,
```

Comment:
why just ping now?

Reply:
We don't sync from the SDK for connect anymore